README.md

```diff
@@ -1,2 +1,138 @@
+
+When the user preforms an action on part of the view, the view should delegate handling that action to the Action Creator. If this is done correctly, the view should only ever have to talk to one other thing, the Action Creator.  For example, if a user clicks on an upload button, the view should tell the Action Creator to create an upload action.
+
+As for inflating view elements with data, the view should only have to listen to one source, the data bus.  Once the view is created, the view should subscribe to events on the data bus.  Once data comes in, the view can then take the data and bind it to whichever view component needs it.
+
+More on the data bus later...
+
+####Action Creator
+The Action Creator has 2 responsibilities.  It's first responsibility is to provide an API for the View to call upon.  These API endpoints should correspond to possible actions the user can preform on the view. If you had a calculator app for example, your Action Creator should have methods such as 'createAddAction', 'createSubtractAction', and so on.  Depending on how you implemented your calculator, those methods may also take a parameter, for example, 'public void createAddAction(float numberToAdd)'.
+
+The Action Creator's second responsibility is to create action objects and post them to the Action Bus.
+
+####Action Bus
+The Action Bus is the event bus that all actions are posted to.  The Stores and Models listen for specific actions posted to this bus in order carry out view logic or business logic, respectively.
+
+####Stores / Models
+Models are where data is stored and transformed.  Models in this case are similar to Models in an MVC type framework.  Models will subscribe to events posted onto the action bus.  To continue with the calculator example, if an AddAction object is posted to the ActionBus, the model will receive that event, then most likely add a number to whatever the current number is then post the updated data to the data bus.
+
+Stores are slightly different than models in that the data stored in the store is strictly related to the current view state.  Unlike Models, what ends up in the view and what ends up in the store is more flexible and up to interpretation.  Our goal is to keep the view as dumb as possible; on the other hand, we don't want to make our app unnecessarily complicated by putting too much in the store.  If, for example, you had a button that just toggled on or off, you might want to just keep that in the view, but if you had a view pager and needed to keep track of which fragment is active, you could put that in the store.
+
+####Data Bus
+The Data Bus is the event bus that all model updates or view data updates are posted to.  The View subscribes to data posted on this bus in order to keep the view in sync with the data.
```

Comment:
When you capitalize "The View", I see this:
![](http://livingcivil.com/wp-content/uploads/2012/01/theview.jpeg)